<?xml version="1.0"?>
<launch>

    <node pkg="aerial_robot_base" type="timestamp_modification" name="timestamp_node"  output="screen" > 
      <param name="ground_truth_sub_name"  value="/mocap/ground_truth/pose"/>
      <param name="estimated_sub_name"  value="/optical_flow/data"/>
      <param name="time_diff"  value="0.4"/>
      <param name="alt_diff"  value="0.04"/>
      
    </node>

</launch>
